### PR TITLE
fix(Tabs): text strings as defaultProps, other refactoring too (Il8n)

### PR DESCRIPTION
**Pull request by hellobrian**
related to #107 

Should have two reviewers on this to review refactoring. Basically, I removed the `const props` variable objects that were being spread across the jsx. Changed the props so that they match with how we're declaring them in our other `render` methods across the lib.

### CODE REVIEW

File: components/Tab.js [hunk 1]
```diff
@@ -14,15 +14,17 @@ class Tab extends React.Component {
     onClick: PropTypes.func,
     onKeyDown: PropTypes.func,
     selected: PropTypes.bool,
-    tabIndex: PropTypes.number,
+    tabIndex: PropTypes.number
   };
 
   static defaultProps = {
+    role: 'presentation',
+    label: 'provide a label',
```
Comment (chrisdhanaraj) [inline components/Tab.js]: we should make the prop required if it's going into defaultProps

**Comment by chrisdhanaraj** [comment]
Yeah that was an odd way of doing props

**Comment by marijohannessen** [comment]
👍 ✅  Looks good to me!

**Comment by chrisdhanaraj** [comment]
👍  ✅

**Comment by chrisdhanaraj** [comment]
Bluemix [toolchain](https://console-regional.ng.bluemix.net/devops/toolchains/6b8b7163-1ec8-4616-82fa-aa467a17af03 "storybook-toolchain-v2"): [Delivery Pipeline](https://console-regional.ng.bluemix.net/devops/pipelines/b84aaac8-8484-4a1e-902a-ebb526eb7998 "storybook-toolchain-v2") deployed [carbon-components-react](https://github.com/carbon-design-system/carbon-components-react/releases/tag/deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170810-155531 "deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170810-155531") to [Bluemix Design System](https://console.bluemix.net/apps/597e3f6e-c208-4e71-97d0-0e4982545e05?env_id=ibm:yp:us-south "ibm:yp:us-south:chrisdhanaraj_org:Bluemix Design System"), including commit a964e890fa00c3729fc645d395d0601e5b02b089

**Pull request merged**